projects/systelab-login/src/lib/README.md

```diff
@@ -54,7 +55,8 @@ This is an example with the three components enabled.
                 [pathTerms]="pathTerms"
                 [pathPrivacy]="pathPrivacy"
                 [txtUsername]="txtUsername"
-                [applicationName]="'App Name'" [moduleName]="'Module'" [version]="'1.4. (build 2012)'" [isLoading]="isLoading">
+                [applicationName]="'App Name'" [moduleName]="'Module'" [version]="'1.4. (build 2012)'" [isLoading]="isLoading"
+                [noticeText}="'Notice Text'">
```

Comment:
Review all the refences of **[notice}** in the README.md files because they have the error to no have the properly closing **bracket]** .